ironfish/src/rpc/adapters/socketAdapter/socketAdapter.ts

```diff
@@ -58,11 +60,13 @@ export abstract class RpcSocketAdapter implements IRpcAdapter {
     port: number,
     logger: Logger = createRootLogger(),
     namespaces: ApiNamespace[],
+    enableAuthentication: boolean,
```

Comment:
I don't think we should take this as a required parameter since we default it in the class